src/test/resources/application-test.yml

```diff
@@ -10,6 +10,8 @@ spring:
     url: 'jdbc:postgresql://localhost:5432/book-a-video-link-test-db'
     username: book-a-video-link
     password: book-a-video-link
+    hikari:
+      minimum-idle: 5
```

Comment:
The inclusion of more integration tests led to running out of connections.  This keeps things under control.